ONSdigital/eq-survey-runner
# Eq 391 pipe an answer

### What is the context of this PR?

This pull request allows an author to specify an alias for a response, and then refer to that response in the text of a later question.

The star wars kitchen sink schema has been modified to demonstrate this functionality
### How to review

1) Check out the branch
2) Run the tests
3) Run the app and go to the dev console
4) Select the star wars survey
5) Answer the question about Chewbaccas age and proceed through the questionnaire
6) On the last page of the questionnaire, verify that there is an additional question asking you to confirm Chewbaccas age and that the value you entered is contained within the question text
### Who worked on the PR

@weapdiv-david worked on this PR so it needs to be reviewed by someone other than him
